opencog/cython/PythonEval.cc

```diff
@@ -575,7 +575,7 @@ PyObject* PythonEval::call_user_function(   const std::string& moduleFunction,
         it != argumentHandles.end(); ++it) {
 
         // Place a Python atom object for this handle into the tuple.
-        PyObject* pyAtom = py_atom(it->value(), pyAtomSpace);
+        PyObject* pyAtom = py_atom((void*) it->atom_ptr_ptr(), pyAtomSpace);
```

Comment:
And this is getting the address of the shared_ptr at the current iteration in the HandleSeq.